src/Admin/IdeasWorkshopQuestionAdmin.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use Doctrine\ORM\QueryBuilder;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+
+class IdeasWorkshopQuestionAdmin extends AbstractAdmin
+{
+    public function createQuery($context = 'list')
+    {
+        /** @var QueryBuilder $proxyQuery */
+        $proxyQuery = parent::createQuery($context);
+
+        $proxyQuery
+            ->leftJoin('o.guideline', 'g')
+            ->orderBy('g.position', 'ASC')
+            ->addOrderBy('o.position', 'ASC')
+        ;
+
+        return $proxyQuery;
+    }
+
+    protected function configureShowFields(ShowMapper $showMapper)
+    {
+        $showMapper
+            ->add('category', null, [
+                'label' => 'Catégorie',
+            ])
+            ->add('name', null, [
+                'label' => 'Nom',
+            ])
+            ->add('position', null, [
+                'label' => 'Position',
+            ])
+            ->add('enabled', null, [
+                'label' => 'Visibilité',
+            ])
+        ;
+    }
+
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
```

Comment:
what about reusing here you QuestionType?
ex:
```
    $builder = $formMapper
        ->getFormBuilder()
        ->getFormFactory()
        ->createBuilder(QuestionType::class);

        $formMapper
            ->add($builder->get('category'))
            ->add($builder->get('name'))
            // ...
        ;
```

https://sonata-project.org/bundles/admin/3-x/doc/reference/form_types.html#adding-a-formbuilderinterface